Formula/sqlite.rb

```diff
@@ -8,7 +8,7 @@ class Sqlite < Formula
 
   livecheck do
     url "https://sqlite.org/news.html"
```

Comment:
```suggestion
    url :homepage
```

The `homepage` redirects to `https://sqlite.org/index.html`, so we would also need to update it (but I can't create a suggestion for it).